apps/prairielearn/src/components/Navbar.html.ts

```diff
@@ -837,12 +837,17 @@ function NavbarInstructor({
 
 function NavbarPublic({ resLocals }: { resLocals: Record<string, any> }) {
   const { course, urlPrefix } = resLocals;
+
+  if (!course) {
+    return '';
+  }
```

Comment:
I have removed this now.

The reason I added it was because I was getting development-specific errors because course wasn't defined in undefined `public` urls. This isn't an issue in production so it makes sense that it should be removed